cli/cli.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+
+"""
+CLI for interfacing with deployed analysis runner.
+See README.md for more information.
+"""
+import os
+import re
+import logging
+import subprocess
+
+import click
+import requests
+import google.auth
+import google.auth.transport.requests
+
+logging.basicConfig(level='INFO')
+
+BRANCH = 'add-cli'
```

Comment:
```suggestion
BRANCH = 'main'
```

Or hardcode it directly in the lookup below.